docs/edge.rst

```diff
@@ -318,19 +318,177 @@ the form.
 
 .. image:: _assets/img/cloud-edge-delete.png
    :alt: CrateDB Edge deletion confirmation screen
+    :scale: 20%
 
 Deleting a custom region does not delete the resources inside that region. To
 also delete the resources inside the region, run the script provided in the
 deletion confirmation screen in your local CLI. This will uninstall CrateDB
 Cloud Edge from your local Kubernetes cluster.
 
 
+.. _edge-tools:
+
+Install CrateDB Edge with third-party tools
```

Comment:
Do we want to call them 'tools'? These are more like Kubernetes distributions... Perhaps "Distribution-specific tutorials"?

Reply:
```suggestion
Install CrateDB Edge using a on-premise k8s distribution.
```